hibernate-core/src/main/java/org/hibernate/query/QueryProducer.java

```diff
@@ -33,11 +33,22 @@
 	 * @return The {@link Query} instance for manipulation and execution
 	 *
 	 * @see jakarta.persistence.EntityManager#createQuery(String)
-	 * @deprecated use {@link #createQuery(String, Class)} or {@link #createStatement(String)}
+	 *
+	 * @deprecated use {@link #createQuery(String, Class)},
+	 * {@link #createUntypedQuery}, {@link #createMutationQuery(String)}
+	 * or {@link #createMutationQuery} depending on intention
```

Comment:
The second `{@link #createMutationQuery}` doesn't mention method parameters. Is this a mistake?

Reply:
Its just an extra link, a mistake in it being there period.  There is only one form of `#createMutationQuery`